Makefile

```diff
@@ -868,7 +868,7 @@ ocamldoc.opt: ocamlc.opt ocamlyacc ocamllex
 	$(MAKE) -C ocamldoc opt.opt
 
 # OCamltest
-ocamltest: ocamlc ocamlyacc ocamllex
+ocamltest: ocamlc ocamlyacc ocamllex otherlibraries
```

Comment:
If Unix isn't being built, then technically ocamltest doesn't need to depend on the otherlibraries target, but I don't think that matters...